.github/workflows/npm-release.yml

```
@@ -0,0 +1,93 @@
+         git commit -m "Bump version to ${NEW_VERSION}" || exit 1
+         git tag "${NEW_VERSION}" || exit 1
+         git push && git push --tags || exit 1
+      - id: node
+        uses: it-at-m/.github/.github/actions/action-npm-build@main
+        with:
+          app-path:  "${{ inputs.app-path }}"
+      - if: "${{ !env.skipDeployment }}"
+        run: npm --prefix ./${{ inputs.app-path }} publish --provenance --access public
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }} # Centralized token in it-at-m GitHub organization
+  build-image:  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)